python/ray/_private/utils.py

```diff
@@ -1107,3 +1109,48 @@ def validate_namespace(namespace: str):
     elif namespace == "":
         raise ValueError("\"\" is not a valid namespace. "
                          "Pass None to not specify a namespace.")
+
+
+def load_certs_from_env():
+    if os.environ.get("RAY_USE_TLS", "0") == "1":
+        with open(os.environ["RAY_TLS_SERVER_CERT"], "rb") as f:
```

Comment:
maybe raise a proper exception if these required environment variables do not exist, otherwise this throws a less useful exception (sure, this is a draft, but keep this comment in mind in case config via env stays)